pkg/webui/components/mobile-menu/index.js

```diff
@@ -50,7 +50,10 @@ const MobileMenu = ({ className, children, user, onItemsClick, onLogout }) => {
       {Boolean(user) && (
         <div className={style.userHeader}>
           <div className={style.userMessage}>
-            <Icon className={style.userIcon} icon="person" nudgeUp />
+            <ProfilePicture
+              profilePicture={user.profilePicture}
```

Comment:
```suggestion
              profilePicture={user.profile_picture}
```

![](https://user-images.githubusercontent.com/5710611/110921971-7419ba00-831f-11eb-8e2f-cbed129119a5.png)